docs/rest_api.md

```diff
@@ -522,46 +522,73 @@ The object will have at least the following fields:
                 - "label", the nic's label.
                 - "macaddr", the nic's mac address.
 		- "networks", a JSON object describing what networks are attached to the nic. The keys are channels and the values are the names of networks attached to those channels.
+        - "port", the port to which the nic is connected to or null if the nic
+          is not connected to any port.
+        - "switch", the switch that has the port to which the nic is connected
+          to or null if the nic is not connected to any port.
```

Comment:
> Need to have the bit about "if the caller is an admin..." in there somewhere.

It's in the [authorization requirements](https://github.com/CCI-MOC/hil/pull/780/files#diff-0d56cc599e156c786c59f0d2bfafb9ddR591) Should we have that in the description too?

> Sorry to keep ping-ponging you...